### PR TITLE
Release config for CD

### PR DESCRIPTION
**Pull request by sukaant**

### CODE REVIEW

File: apps/estuary/config/integration.exs [hunk 1]
```diff
@@ -9,6 +9,7 @@ config :prestige,
   ]
 
 config :estuary,
+  elsa_endpoint: [localhost: 9092],
   divo: "docker-compose.yml",
```
Comment (dunson062786) [inline apps/estuary/config/integration.exs]: Is this still correct? Is this what it is on aws?

Comment (sukaant) [reply apps/estuary/config/integration.exs]: This is the integration config

File: apps/estuary/lib/estuary/application.ex [hunk 2]
```diff
@@ -41,9 +37,18 @@ defmodule Estuary.Application do
   end
 
   defp validate_topic_exists do
-    case Elsa.Topic.exists?(@elsa_endpoint, @event_stream_topic) do
-      true -> :ok
-      false -> Elsa.Topic.create(@elsa_endpoint, @event_stream_topic)
+    case Elsa.Topic.exists?(
```
Comment (ManApart) [inline apps/estuary/lib/estuary/application.ex]: Personally I think it'd be cleaner to extract these guys into local variables (just for readability).

Comment (NathanielScottStevens) [reply apps/estuary/lib/estuary/application.ex]: I think the way we've solved that before is by throwing them in functions.

**Comment by dunson062786** [comment]
Is there anything else that this pull request does than move elsa endpoint from the main config to the integration config.

File: apps/estuary/lib/estuary/application.ex [hunk 3]
```diff
@@ -2,14 +2,10 @@ defmodule Estuary.Application do
   # See https://hexdocs.pm/elixir/Application.html
   # for more information on OTP Applications
   @moduledoc false
-
   use Application
 
   alias Estuary.EventTable
 
-  @elsa_endpoint Application.get_env(:estuary, :elsa_endpoint)
```
Comment (jmorris-pillar) [inline apps/estuary/lib/estuary/application.ex]: you found the footgun :-)

File: apps/estuary/lib/estuary/application.ex (hunk 2 above)
Comment (jmorris-pillar) [reply apps/estuary/lib/estuary/application.ex]: Agree^ . This is being done in more than 1 place so they should be in functions

File: apps/estuary/lib/estuary/event_table.ex [hunk 4]
```diff
@@ -3,11 +3,9 @@ defmodule Estuary.EventTable do
   This module will create event_stream table and insert data in the table
   """
 
-  @event_stream_table_name Application.get_env(:estuary, :event_stream_table_name)
-
   def create_table do
     Prestige.execute(
-      "CREATE TABLE IF NOT EXISTS #{@event_stream_table_name} (author varchar, create_ts bigint, data varchar, type varchar)"
+      "CREATE TABLE IF NOT EXISTS #{Application.get_env(:estuary, :event_stream_table_name)} (author varchar, create_ts bigint, data varchar, type varchar)"
```
Comment (jmorris-pillar) [inline apps/estuary/lib/estuary/event_table.ex]: toss this in a local function imo

**Pull request merged**